src/tasks/selfSell.ts

```diff
@@ -909,6 +905,8 @@ const setupSelfSellTask: () => void = () =>
           throw new Error("Order validity too large");
         }
 
+        const tokens = (await getTokensWithBalanceAbove(minValue, settlementDeployment.address)).filter((token) => token != toToken);
```

Comment:
I think this should also filter out `WETH` (which happens to be the `toToken` in practice but making that explicit might be a good idea). There is a technical reason for why `WETH` withdraws cause issues with that payout script but I was not able to find it right now.

Reply:
It's because we cannot sell WETH for WETH (theoretically we should make ETH the toToken to also avoid unrwapping later on). Will add a comment...